css/properties.json

```diff
@@ -679,6 +679,7 @@
     "media": "visual",
```

Comment:
According to the spec. the syntax of `-webkit-text-stroke` actually also needs to be changed to `"&lt;line-width&gt; || &lt;color&gt;"`.

Sebastian